packages/web/components/discover/discover-filter-from.component.tsx

```diff
@@ -42,6 +45,7 @@ export function DiscoverFilterFormComponent(
       <DiscoverFilterSectionComponent title="Language">
         <Form.Item key="originLanguage" name="originLanguage">
           <Select
+            allowClear
```

Comment:
You suggested using multiple select, but because tmbd origin language is a string, I ignored using multi select to not dealing with having an array, now we can remove language from query by click on remove search